app/services/doc_auth/acuant/pii_from_doc.rb

```diff
@@ -19,7 +20,7 @@ class PiiFromDoc
 
       def initialize(id_data_fields)
         @name_to_value = {}
-        id_data_fields['Fields'].each do |field|
+        id_data_fields['Fields']&.each do |field|
```

Comment:
not 100% sure I understand why the `&` is on the outside of the loop like this? I'd expect if the field is sometimes there, then the optional stuff would be inside the loop?

Reply:
Yeah, I put this in because we have some tests that don't specify `id_data_fields['Fields']`, but I think it'd be better just to update the tests than change this